api/apps/api/src/modules/authentication/authentication.controller.ts

```diff
@@ -27,13 +27,16 @@ import { SignUpDto } from './dto/sign-up.dto';
 import { UserAccountValidationDTO } from './dto/user-account.validation.dto';
 import { LocalAuthGuard } from './local-auth.guard';
 import { IsMissingAclImplementation } from '@marxan-api/decorators/acl.decorator';
+import { Throttle, ThrottlerGuard } from '@nestjs/throttler';
 
 @Controller('/auth')
 @ApiTags('Authentication')
 export class AuthenticationController {
   constructor(private readonly authenticationService: AuthenticationService) {}
 
   @UseGuards(LocalAuthGuard)
+  @UseGuards(ThrottlerGuard)
+  @Throttle(25, 60)
```

Comment:
I think we can throttle most of/all the endpoints in this controller: sign-up+verification, if automated, can facilitate mass bot signup (though even at 25signups per minute, we can get 36k bots signed up per day 😢, so that's a lost battle in any case).

No real need to throttle jwt refresh and sign-out, though.

But maybe we can move the guard and the throttle setting to the controller (rather than the `sign-in` endpoint handler) and this way each endpoint handler will have a 25rpm limit.

Reply:
or apply the guard and throttle setting to `sign-up` too, leaving the other endpoints alone